docs/tutorials/gettingstartedexample/gse-gen-basic.md

```diff
@@ -28,59 +28,67 @@ A supported interface module on a PXI system running Windows.
 
 ## Initial Hardware Setup
 
-1. Complete installation of hardware as described in the Getting Started Guide linked above.
-2. No physical connections to other modules or devices are required to complete the basic tutorial.
+Complete installation of hardware as described in the Getting Started Guide for your interface module. No physical connections to other modules or devices are required to complete this tutorial.
 
 ## Initial Software Setup
 
-1.  First complete installation of software specified in the Getting Started Guide linked above.
-2.  Use the NI Example Finder to create a default project for your specific interface module.
-    - Open LabVIEW and click **Help -> Find Examples...**
+1.  Complete installation of software specified in the Getting Started Guide for your interface module.
+2.  Use the NI Example Finder to create a default project for your interface module.
+    1. Open LabVIEW and click **Help » Find Examples...**
```

Comment:
On the rendered page I don't see these deeper sub-bullets rendered as 'a' 'b' 'c' but just 'a' 'a' 'a'.  I think this is because the markdown translation struggles with recognizing it as a continuous list when there are images inserted between each list item.  Sometimes playing with the indentation of the image will 'fix' the issue, but I don't know of a great solution to this situation. I think this might be why it was an unordered list originally instead of an ordered list.

Reply:
FYI, I am looking here:
https://dharlach-ni.github.io/automotive-camera-module-reference/tutorials/gettingstartedexample/gse-gen-basic.html

Reply:
I restructured that section a bit to avoid the need to have a nested list like that. I see the PR has updated with the new commit, so I think things are taken care of, from what I can tell.